### DISCUSSION TITLE
Need Assistance with the new Forked updated version

### DISCUSSION THREAD

**Discussion by SlayerMarth** [ORelio/Minecraft-Console-Client#2121] [Q&A] [answered]
Hello,

If this isn't where this goes, I apologize. I never use Github so Im new here. 
I needed some help with the new fork of this. For some reason, every single version after [20220819-1](https://github.com/milutinke/Minecraft-Console-Client/releases/tag/20220819-1) doesn't work for me. I've downloaded each updated version of it and the usual icon doesn't appear, nor will the .exe open. I double click them, nothing happens. Whereas that version, 20220819-1, the usual icon is there and it opens. But it doesn't support 1.19.1, which is what Im looking for. Any ideas? 

Everything on my end is up to date in terms of .net framework etc.

**Comment by breadbyte** [comment, marked as answer]
You will need the latest version of [.NET](https://dotnet.microsoft.com/en-us/download), .NET 6.

This is different from .NET Framework. We have recently migrated from .NET Framework to the latest .NET version.

**Reply by SlayerMarth** [reply]
Gotcha, that makes sense. Looks to be working now, although now when I login, it just says connecting to Minecraft.net, then Not connected to any server. Use /help for help. Am I doing something else wrong? Went through the documentation and followed everything, unless I overlooked something. Sorry if all this is literally just me being dumb

**Reply by breadbyte** [reply]
Try deleting your `SessionCache.ini` and then login again.

**Reply by milutinke** [reply]
Note that 1.19.1 is not yet supported.

**Reply by SlayerMarth** [reply]
> Try deleting your `SessionCache.ini` and then login again.

Sorry for the delayed response, some things got hectic for me. I don't have a SessionCache.ini anywhere to delete. Should I?  This is the first time I'm re-downloading this in about 2 years and have a new PC and all, so it was a fresh install.

**Reply by milutinke** [reply]
> > Try deleting your `SessionCache.ini` and then login again.
> 
> Sorry for the delayed response, some things got hectic for me. I don't have a SessionCache.ini anywhere to delete. Should I? This is the first time I'm re-downloading this in about 2 years and have a new PC and all, so it was a fresh install.

It is created by the program when you log in for the first time, and it's auto refreshed afterwards, unless you change the account.
I suggesting completely deleting .NET, and then installing fresh .NET 6 and then downloading the latest build of MCC and running it.

**Reply by SlayerMarth** [reply]
> It is created by the program when you log in for the first time, and it's auto refreshed afterwards, unless you change the account. I suggesting completely deleting .NET, and then installing fresh .NET 6 and then downloading the latest build of MCC and running it.

That fixed it, not sure why it didn't when I'd done it initially, weird lol but all works perfectly now. Thank you!
